CONTRIBUTING.md

```diff
@@ -1,32 +1,167 @@
+        - The Student guide's Challenge markdown files
+        - `/Resources` 
+            - The code and supporting files the students will need throughout the hack.
+1. Re-Read the [What The Hack Author's Guide](./000-HowToHack/WTH-HowToAuthorAHack.md) (seriously) and make sure your hack follows the templates & styles for consistency.
+
+### Release Process
+
+When you feel your hack is finished and ready for release, this is the process we will follow:
+
+1.	The WTH team will assign your new hack a number.	
+1.	You should immediately rename your root folder to use that number. (i.e. "`XXX-MyAwesomeHack`" to "`067-MyAwesomeHack`")
+1.	The WTH team will schedule a 60-minute "pre-PR review" meeting with you and any co-authors. 
+    - The purpose of this meeting is to go through the content together and reduce the amount of back and forth review cycles on Github once your Pull Request is submitted.
+    - During this review, the WTH team will go through the text with a fine-toothed comb checking for:
+        - Adherence to the [WTH Author's Guide](./000-HowToHack/WTH-HowToAuthorAHack.md)
+        - All links work, especially the navigation links
+        - There are no links to the WTH repo or Coach's guide from the Student guide! (See the [WTH Author's Guide](./000-HowToHack/WTH-HowToAuthorAHack.md))
+        - All images show properly.
+        - Any syntax, grammar or punctuation problems that the reviewers see and want you to address.
+        - This is NOT a technical content review. As the author(s), YOU are the subject matter experts. The WTH team will trust that you have taken care of the technical bits.
+    - **NOTE:** It is important that you take notes through-out the meeting so that you can go away, make any changes requested, and not miss anything.
+1.	Once you have completed any requested changes from the "pre-PR review", you can submit a pull request to the WTH repo.
+1.	The WTH team will review your PR and leave comments if there are any requested changes that still remain. If there are requested changes, please add further comments if you have clarifying questions to ask, or arguments against, the requested changes (that’s ok).
+    - **NOTE:** Make any requested changes by continuing to commit to your fork. The PR will automatically update with your changes.  You do NOT need to create a new pull request!
+1.	Once you have addressed any requested changes from the WTH team, the WTH team will accept and merge the PR.
+
+### Use work in progress PRs for early feedback
+
+If you choose not to collaborate with the WTH team via Microsoft Teams, alternatively you can use a work-in-progress Pull Request.
+
+A good way to communicate before investing too much time is to create a "Work-in-progress" PR and share it with the WTH team. The standard way of doing this is to add a "[WIP]" prefix in your PR's title and assign the **do-not-merge** label. This will let people looking at your PR know that it is not well baked yet.
```

Comment:
Out of curiosity why **do-not-merge** label instead of making it a draft PR, which makes it not mergeable?

Reply:
Idk?  This was borrowed text from the Dapr project. :)

Reply:
Outdated as we have since discussed draft vs. regular PR.